### PR TITLE
DOC-3754: Added instructions for upgrading a node to a minor version.

### PR DESCRIPTION
**Pull request by taylorharvey-r3**

### CODE REVIEW

File: content/en/platform/corda/4.8/enterprise/minor-version-node-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+date: '2021-08-02'
```
Comment (michelemarsden) [inline content/en/platform/corda/4.8/enterprise/minor-version-node-upgrade.md:2]:
```suggestion
date: '2021-12-21'
```

File: content/en/platform/corda/4.8/enterprise/minor-version-node-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+{{< note >}}
+The protocol tolerates node outages. Peers on the network wait for your node to become available after upgrading.
+{{< /note >}}
+
+## Step 1: Drain the node
+
+Node operators must drain nodes (or CorDapps on nodes) before they can upgrade them. Draining brings all [flows](cordapps/api-flows.md) that are currently running to a smooth halt. The node finishes any work already in progress, and queues any new work. This process frees CorDapps from the requirement to migrate workflows from an arbitrary point to another arbitrary point—a task that would rapidly become unfeasible as workflow
+and protocol complexity increases.
+
+To drain a node, run `gracefulShutdown`. This waits for the all currently running flows to be completed and then shuts the node down.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.8/enterprise/minor-version-node-upgrade.md:36]:
```suggestion
To drain a node, run `gracefulShutdown`. This waits for all running flows to be completed and then shuts the node down.
```

File: content/en/platform/corda/4.8/enterprise/minor-version-node-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+the CorDapps are well-written and the required counterparties are online, drains may only take a few seconds.
+
+For a smooth node draining process, avoid long-running flows.
+{{< /warning >}}
+
+## Step 2: Replace `corda.jar` with the new version
+
+Replace the `corda.jar` with the latest version of Corda.
+
+Download the latest version of Corda from [our Artifactory site](https://software.r3.com/artifactory/webapp/#/artifacts/browse/simple/General/corda/net/corda/corda-node).
```
Comment (michelemarsden) [inline content/en/platform/corda/4.8/enterprise/minor-version-node-upgrade.md:50]:
```suggestion
Download the latest version of Corda from [Artifactory](https://software.r3.com/artifactory/webapp/#/artifacts/browse/simple/General/corda/net/corda/corda-node).
```

File: content/en/platform/corda/4.8/enterprise/minor-version-node-upgrade.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+{{< important >}}
+Corda 4 requires Java 8u171 or any higher Java 8 patch level. Java 9+ is not currently supported.
+{{< /important >}}
+
+## Step 3: Start the node
+
+Start the node in the normal way.
+
+The node will perform any automatic data migrations required, which may take some
```
Comment (michelemarsden) [inline content/en/platform/corda/4.8/enterprise/minor-version-node-upgrade.md:62]:
```suggestion
The node performs any required automatic data migrations, which may take some
```

File: content/en/platform/corda/4.8/enterprise/minor-version-node-upgrade.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+Run this command in the shell:
+
+`run setFlowsDrainingModeEnabled enabled: false`
+
+Your upgrade is complete.
+
+## Notes
+
+{{< warning >}}
+You must align the multi-RPC client version with the node version. That means that both must be running the same version of Corda Enterprise. See [Querying flow data](node/operating/querying-flow-data.md) for more information.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.8/enterprise/minor-version-node-upgrade.md:76]:
```suggestion
You must align the multi-RPC client version with the node version. Both must be running the same version of Corda Enterprise. See [Querying flow data](node/operating/querying-flow-data.md) for more information.
```

**Review by michelemarsden — COMMENTED** [review]
Please see my comments.

**Review by adelel1 — APPROVED** [review]
LGTM

**Review by michelemarsden — APPROVED** [review]
LGTM

**Comment by peterli-r3** [comment]
Does this link work?
https://software.r3.com/artifactory/webapp/#/artifacts/browse/simple/General/corda/net/corda/corda-node
It shows blank page. You gotta search for Corda. 
I use this link: https://software.r3.com/ui/packages/gav:%2F%2Fnet.corda:corda

**Comment by taylorharvey-r3** [comment]
@peterli-r3 Good spot. I'll substitute with your link. Thanks!

**Pull request merged**